app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.includes(:actors).find(params[:id])
+    @actors = @movie.actors.order(age: :asc)
+    @average_age = @actors.average(:age)
+  end
+
+  def update
```

Comment:
To add an actor to a movie, we are actually creating an joins record that associates an existing movie to an existing actor. In this sense, we're actually *creating* something (an ActorMovie) rather than *updating* the movie resources. So, a more RESTful option here would be to route the form in your view to a `POST` to `/movies/:movie_id/actors` and that can direct to a joins controller: `ActorMoviesController#create`

In this create action, you could find the existing movie and actor from the params, and then create a ActorMovie object, and redirect back to the movie show page.